fixes lavaland_surface_xeno_nest

spockye opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
4 missing tiles at the entrance of the hive

![2023 01 13-17 43 31](https://user-images.githubusercontent.com/79304582/212373380-92b1540d-64f5-4f32-b306-51f1076b5261.png)

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

<!-- Tick the box below (put an X instead of a space between the brackets) if you have tested your changes and this is ready for review. Leave unticked if you have yet to test your changes and this is not ready for review. -->

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
mapping errors = bad!!
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
fix: fixes missing tiles on xeno lava nest
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

spockye committed:
fix

4 missing tiles

Erikafox commented:
does this solve the issue where every enemy inside the ruin dies while it's spawning?

spockye committed:
drastic devious EVILLL change

changes all floors within the  ruin to normal basalt so they don't boil

#savebenolives

Latentish commented:
idk man works fine on my machine
